compiler/src/iree/compiler/Codegen/Passes.h

```diff
@@ -504,6 +504,15 @@ void buildSPIRVCodegenPassPipeline(OpPassManager &pm, bool enableFastMath);
 // at the bufferized linalg level.
 std::unique_ptr<Pass> createVMVXLowerLinalgMicrokernelsPass();
 
+//------------------------------------------------------------------------------
```

Comment:
I am hoping this is annoying enough for you that you will volunteer to break this file into individual `Passes.h` files for each backend :P